common/src/main/java/org/apache/rocketmq/common/stats/StatsItemSet.java

```diff
@@ -158,6 +158,43 @@ public void addValue(final String statsKey, final int incValue, final int incTim
         statsItem.getTimes().addAndGet(incTimes);
     }
 
+    public void delValue(final String statsKey) {
+        StatsItem statsItem = this.statsItemTable.get(statsKey);
+        if (null != statsItem) {
+            this.statsItemTable.remove(statsKey);
+        }
+    }
+
+    public void delValueByFirstKey(final String statsKey, String separator) {
```

Comment:
I thinks the method name "delValueByFirstKey" may not be suitable.And may be you can rename it, such as "delValueByPrefixKey";

Reply:
I have renamed all.